tools/stress-cluster/services/Stress.Watcher/publish.ps1

```diff
@@ -1,14 +1,8 @@
 param(
-    [string] $Tag = "latest",
-    [bool] $Login = $true
+    [string] $Tag = "latest"
 )
 
-docker build . -t stresswatcher
-docker tag stresswatcher stresstestregistry.azurecr.io/services/stresswatcher:$Tag
-docker tag stresswatcher stressprodregistry.azurecr.io/services/stresswatcher:$Tag
+docker build . -t azsdkengsys.azurecr.io/stress/stresswatcher:$Tag
```

Comment:
Since we're using one common place, let's delete this file and just use the pipeline that can publish after merge (and for local testing we can just write our own script): https://github.com/Azure/azure-sdk-tools/blob/main/eng/containers/ci.yml